MailChimp.Net/Configuration/MailChimpConfiguration.cs

```diff
@@ -1,17 +1,36 @@
 ﻿using System;
-using System.Configuration;
+using System.Linq;
 using MailChimp.Net.Interfaces;
+using Microsoft.Extensions.Configuration;
 
 namespace MailChimp.Net
 {
     public class MailChimpConfiguration : IMailChimpConfiguration
     {
-        public static int DefaultLimit => Common.DefaultLimit;
+        public MailChimpConfiguration()
+        {
+            var builder = new ConfigurationBuilder()
```

Comment:
Shouldn't this class be taken from the constructor of the mailchimp configuration? 
It would make more sense to have a ```MailChimpOptions``` class that you inject by ctor injection.